test/common/index.js

```diff
@@ -263,8 +263,8 @@ function createZeroFilledFile(filename) {
 
 
 const pwdCommand = isWindows ?
-  ['cmd.exe', ['/d', '/c', 'cd']] :
-  ['pwd', []];
+  [new URL('file:///C:/Windows/system32/cmd.exe'), ['/d', '/c', 'cd']] :
+  [new URL('file:///bin/pwd'), []];
```

Comment:
Why these changes? Are we sure these absolute paths are safe assumptions?

Reply:
Not 100% sure and I'd like to see if it's not the case on any platform covered by CI first.

Then it will most likely be moved to separate `test-child-process-urlfile.mjs` test.

If there is a concise&robust way to get absolute url or absolute path to `pwd` or any other convenient good-for-test executable, or if we can create one in fixtures, I'd like to know.

Reply:
I would guess the "where"/"which" commands should generally be available, even outside of CI since tests should work locally too.